src/nft/components/explore/Carousel.tsx

```diff
@@ -37,36 +32,32 @@ const CarouselCardContainer = styled.div`
 
 const CarouselItemCard = styled(a.div)`
   ${carouselHeightStyle}
-  ${carouselItemStyle}
 
   display: flex;
   justify-content: center;
-  padding: 3px 32px 32px 32px;
-
+  padding: 4px 12px 32px;
   position: absolute;
   will-change: transform;
+
+  @media screen and (min-width: ${({ theme }) => theme.breakpoint.lg}px) {
+    padding: 4px 32px 32px;
+  }
 `
 
 const CarouselItemIcon = styled.div`
   ${carouselHeightStyle}
-  ${carouselItemStyle}
-
-  display: flex;
   align-items: center;
-  padding-left: 8px;
-  padding-right: 8px;
+  color: ${({ theme }) => theme.textPrimary};
   cursor: pointer;
+  display: none;
+  padding-left: 4px 8px 32px;
   user-select: none;
-
-  color: ${({ theme }) => theme.textPrimary};
-
+  @media only screen and (min-width: ${({ theme }) => `${theme.breakpoint.sm}px`}) {
```

Comment:
Any particular reason for this change? I think conceptually the original version was a bit cleaner with `display: flex` being default and `display: none` override for small devices (only).

Reply:
following the mobile-first media query pattern w/ min-width overrides

Reply:
i don't think it matters but i would suggest not switching the pattern for existing components as it makes prs harder to read. at least until we decide on a path forwards as a team.